### ISSUE TITLE
[JavaScript] Highlighting breaks when importing multiple members from a module

### ISSUE THREAD

**Issue by ccampbell** [sublimehq/Packages#186]
Noticed this after the changes in 409ccb469bca8bcc2a5c70c562d1db3428d3015f

Example code

```javascript
import {something, somethingElse} from "somewhere";
import {anotherthing as another} from "anotherplace";
import Hello from "somewhereelse";

class Whatever {
    constructor() {
    
    }
}
```

**Before changes**

![](https://devi.vimeocdn.com/craig/iFbI)

**After changes**

![](https://devi.vimeocdn.com/craig/APnA)

**Comment by ccampbell** [comment]
This may be related to #185. Rather than making `import` and `from` keywords, it may make sense to make actual statements for them.  You can find good documentation/examples here:

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

It is probably also broken for exports

**Comment by wbond** [comment]
Excellent, I'll get changes made to support this syntax.

**Comment by wbond** [comment]
This should be resolved by 412635a10fe39ae507656d6aee7e502180de2505 and c162e39f665547f0e2537ff053fec422e380c603.
